src/node/internal/http_response.js

```diff
@@ -0,0 +1,66 @@
+import {inherits} from 'node-internal:http_util';
```

Comment:
What is the provenance of this impl? Did you adopt this directly from Node.js or from a polyfill? There have been a number of critical bug fixes and improvements in the Node.js implementation very recently that we want to make sure we bring in and the various polyfills that exist out there do not yet include those fixes.